### PR TITLE
Add asc/desc icons for sortable table

### PR DESCRIPTION
**Pull request by pseuyi**

### CODE REVIEW

File: src/icons/downArrow.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const icon = '<svg width="8px" height="5px" viewBox="0 0 8 5" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs></defs><g id="Sortable-headers" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="Nearby-inventory" transform="translate(-680.000000, -249.000000)" fill="#252729"><polygon id="ic_downarrow" transform="translate(684.000000, 251.500000) scale(1, -1) translate(-684.000000, -251.500000) " points="684 249 688 254 680 254"></polygon></g></g></svg>';
```
Comment (kangax) [inline src/icons/downArrow.js]: It looks like we're duplicating icon content (svg markup) in this .js file and in .svg file below (as well as for an up arrow). Do we need svg file then? Can we only have one?

File: src/components/Table/Table.jsx [hunk 2]
```diff
@@ -67,7 +70,7 @@ class Table extends React.Component {
     const tdElems = this.table.getElementsByTagName('td');
     forEach(tdElems, (td) => {
       td.removeEventListener('mouseenter', this.handleColumnMouseEnter);
-      td.removeEventListener('mouseleav', this.handleColumnMouseLeave);
+      td.removeEventListener('mouseleave', this.handleColumnMouseLeave);
```
Comment (kangax) [inline src/components/Table/Table.jsx]: wow, good catch

File: src/components/Table/Table.jsx [hunk 3]
```diff
@@ -136,10 +139,17 @@ class Table extends React.Component {
     return this.renderHeader({ sticky: true });
   }
 
+  renderCarat() {
+    return this.props.sort.order === 'asc' ?
+      <Icon color="#fff" icon={downArrow} />
+      : <Icon color="#fff" icon={upArrow} />;
```
Comment (kangax) [inline src/components/Table/Table.jsx]: maybe combine? 

`return <Icon color="#fff" icon={this.props.sort.order === 'asc' ? downArrow : upArrow} />`

File: src/icons/downArrow.js (hunk 1 above)
Comment (pseuyi) [inline src/icons/downArrow.js]: you're right dont need the .svg

File: src/components/Table/Table.jsx (hunk 2 above)
Comment (yale) [inline src/components/Table/Table.jsx]: these are a lot of event listeners....

Comment (pseuyi) [inline src/components/Table/Table.jsx]: @yale these are not being used at all at the moment!

File: src/components/Table/style.scss [hunk 4]
```diff
@@ -43,12 +43,13 @@
     th {
       padding-top: 0;
       padding-bottom: 0;
-      font-weight: bold;
+      // font-weight: bold;
```
Comment (yale) [inline src/components/Table/style.scss]: delete this line

File: src/styles/icons.scss [hunk 5]
```diff
@@ -1,8 +1,8 @@
 %icon--overflow {
   &:before {
     content: url('data:image/svg+xml;utf8,<svg width="30px" height="30px" viewBox="0 0 30 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch --><defs></defs><g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="Group-Copy" transform="translate(13.000000, 7.000000)" fill="#575757"><circle id="Oval" cx="2" cy="2" r="2"></circle><circle id="Oval-Copy" cx="2" cy="8" r="2"></circle><circle id="Oval-Copy-2" cx="2" cy="14" r="2"></circle></g></g></svg>');
-    display: block;
+    // display: block;
```
Comment (yale) [inline src/styles/icons.scss]: delete this

Comment (pseuyi) [inline src/styles/icons.scss]: haha *scream emoji*

**Pull request merged**